### PR TITLE
fix component imports and rename track event

### LINKED ISSUE #2151: Rename TrackEvent for generic components
rename TrackEvent to something non-analytics specific, to keep relationships well decoupled

### PR DESCRIPTION
**Pull request by mparke**
closes #2151

### CODE REVIEW

File: components/modals/DeleteModalBody.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { PropTypes } from 'react';
 import ConfirmModalBody from './ConfirmModalBody';
 import { ScrollingList } from '../lists';
-import { TrackEvent } from '~/actions/trackEvent.js';
+import { default as EmitEvent } from '../utils';
```
Comment (eatonphil) [inline components/modals/DeleteModalBody.js]: `import EmitEvent from '../utils';`?

File: components/forms/Select.js [hunk 2]
```diff
@@ -13,7 +13,12 @@ export default function Select(props) {
         disabled={props.disabled}
         onChange={(e) => {
           const value = String(e.target.value);
-          TrackEvent('Select', value.indexOf(':') < 0 ? value.split(':')[0] : value, props.name);
+          EmitEvent(
```
Comment (eatonphil) [inline components/forms/Select.js]: Would be easier to read than this if it took a dict. nbd

File: src/analytics.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    window.ga('send', 'event', {
+      category: cate,
+      action: action,
+      label: label,
+      value: value,
+    });
+  }, false);
+}
+
+function init(environment, GA_ID) {
```
Comment (eatonphil) [inline src/analytics.js]: This is a global constant defined in `src/constants`. I would just import it from here instead of passing it in...

Comment (eatonphil) [inline src/analytics.js]: I guess the environment is the same way.

**Comment by eatonphil** [comment]
Code looks good. Failing tests though. Pulling locally to test.

**Comment by mparke** [comment]
@eatonphil s/b fixed

**Pull request merged**